preprocessing/nextclade/tests/factory_methods.py

```diff
@@ -12,23 +12,31 @@
 
 
 @dataclass
-class TestCase:
+class ProcessingTestCase:
     name: str
     input: UnprocessedEntry
     expected_output: ProcessedEntry
 
+
 @dataclass
 class UnprocessedEntryFactory:
     _counter: int = 0
```

Comment:
As we anyways only use this Factory for the tests below let's just get rid of the _counter? As you modified the code to now pass accessions its not really needed anymore. We also don't need reset_counter and to create a fixture for it, so will save some code :-)

Reply:
I just added the changes in a commit to move this forward